### ISSUE TITLE
Allow flashing via FTDI adapter

### ISSUE THREAD

**Issue by stylesuxx** [stylesuxx/esc-configurator#37]
This would allow ESC's to be flashed without a need for a flight controller as suggested by @AlkaMotors

One could simply use a FTDI adapter or arduino or anything similar.

**Comment by stylesuxx** [comment]
This [library](https://github.com/dword1511/onewire-over-uart) allows to make a one wire interface from an Arduino and also shows how to convert a UART/FTDI adapter into a one wire interface.

**Comment by knoopx** [comment]
I definitely need this for AM32, the configurator has this "direct connection" option but works on windows/linux only.

**Comment by khari05** [comment]
Just wondering, is there any progress on this feature? I'm not sure if this is helpful but the original BLHeLiSuite supported flashing an Arduino with the correct firmware to interface over serial. A first step could just be adding support to use that same protocol to talk to an ESC after someone already sets up communication through BLHeLiSuite.

**Comment by stylesuxx** [comment]
Nope, no progress on this one. It's a hard sell for me to be honest. I would need to dump a lot of time into this feature and I simply don't see the use case. FCs are relatively cheap, so if you buy an Arduino or an FC makes almost no difference. I know Alka is working on some additional bootloader functionality to integrate a 4 way interface. Once this is done I might look into it to at least allow AM32 being flashed via FTDI.

The Arduino 4 way interfaces I have seen all don't seem to work straight forward. And I can't really use the one that BLHeli ships with.

But by any means, if anyone wants to integrate this, I'd be more than happy to merge a PR.

**Comment by khari05** [comment]
If anyone else finds this issue, the workaround I used was installing betaflight on an STM32 dev board and using that as a temporary flight controller that could talk to bluejay configurator
